Python_env/hurrevac_storms.py

```diff
@@ -95,12 +95,14 @@ def GetStormsJSON(self):
 ##        logging.debug("Running GetStormsJSON")
         manage = Manage()
         manage.handleProxy()
-        openUrl = requests.get(hurrevacSettings['HurrevacStormsURL'])
+        openUrl = requests.get(hurrevacSettings['HurrevacStormsURL'], timeout=10)
```

Comment:
Why only 10 seconds? Is that enough time?

Reply:
Ten seconds should be enough time as the data is very small. Added code to try three times with two seconds in between each before raising an error.